client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/base/BaseScreenModel.kt

```diff
@@ -52,8 +54,13 @@ abstract class BaseScreenModel<S, E>(initialState: S) : StateScreenModel<S>(init
             try {
                 callee()
             } catch (exception: Exception) {
-                when(exception){
-                    is InvalidCredentialsException -> onError(ErrorState.InvalidCredentials(exception.message.toString()))
+                when (exception) {
```

Comment:
no need to use `when` the catch is enough
use more one  catch and remove `when `
this is what bareq said in old reviews

Reply:
@Asiasama710 i'm adding when to add limit